client/src/graphics/draw.rs

```diff
@@ -340,6 +379,10 @@ impl Draw {
         );
         timestamp_index += 1;
 
+        self.yakui_vulkan.transfers_finished(&yakui_vulkan_context);
```

Comment:
I think this morally belongs at the end of `fn wait`.

Reply:
That does make a lot of sense.